tests/ergo_run_test.go

```diff
@@ -542,3 +551,223 @@ func TestRunOSX(t *testing.T) {
 		t.Fatalf("Expected \"ergo test response\" as response and got: %s\r\n", rez)
 	}
 }
+
+func TestConfigDynamicWindows(t *testing.T) {
+	if *getOS() != "windows" {
+		t.Skip("Not running windows run specific tests")
+	}
+	err := setupErgo()
+
+	if err != nil {
+		t.Fatalf("Could not perform setup for windows. Got %s", err.Error())
+	}
+
+	defer cleanSetup()
+	//start a web server that will initialy have no mapping in our proxy
+	startDynamicTestWebServer()
+
+	//run ergo
```

Comment:
I would remove this comment, the code below  `ergo('run')` is pretty clear

Reply:
👍